sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -412,6 +491,7 @@ def convert_feature(fx):
         kind=kind,
         subsample=subsample,
         random_state=random_state,
+        is_categorical=is_categorical,
```

Comment:
I think that it would make more sense here to pass `categorical_features` and to create `is_categorical` in the class if it is necessary. The thing is that we are going to deprecate `plot_partial_dependence` in the favor of some class methods that will use the display and we will then require to have the `categorical_features` there.